docs/technical_reference/unit_models/ion_exchange_0D.rst

```diff
@@ -0,0 +1,317 @@
+Ion Exchange 0D
+===============
+
+.. index::
+   pair: watertap.unit_models.ion_exchange_0D;ion_exchange_0D
+
+The main assumptions of the implemented model are as follows:
+
+1) Model dimensionality is limited to a 0D control volume
+2) Single liquid phase only
+3) Steady state only
+4) Single solute and single solvent (water) only
+5) Plug flow conditions
+6) Isothermal conditions
+7) Constant separation-factor treatment with favorable Langmuir isotherm
+
+Introduction
+------------
+
+Ion exchange is the reversible transfer of one or more solutes between a fluid phase and a sorbent.
+This process is becoming increasingly popular in drinking water treatment applications where it is
+used for water softening and demineralization. This implementation of the fixed-bed ion exchange model
+accounts for process equilibrium, kinetics, and hydrodynamics to predict performance, bed and column geometry, and capital/operating costs.
+The ion exchange process operates as a cycle with four steps: (1) service, i.e., treatment, (2) backwashing, (3) regeneration, (4) rinsing.
+Critical to predicting performance of an ion exchange process is having an estimate for the breakthrough time,
+or the duration of treatment before the solute begins exiting the column at a concentration unacceptable to the operator.
+At this time, the mass transfer zone is approaching the end of the ion exchange bed, the resin is nearing exhaustion,
+and the regeneration cycle can begin. Fundamental to this model is the assumption that the isotherm between the solute
+and the resin is favorable, and thus the mass transfer zone is narrow.
+
+
+Ports
+-----
+
+The model provides three ports (Pyomo notation in parenthesis):
+
+* Inlet port (inlet)
+* Outlet port (outlet)
+* Regen port (regen)
+
+Sets
+----
+
+.. csv-table::
+   :header: "Description", "Symbol", "Index"
+
+   "Time", ":math:`t`", "[0]"
+   "Phases", ":math:`p`", "['Liq']"
+   "Components", ":math:`j`", "['H2O', 'Na_+', 'Ca_2+', '\Cl_-', 'Mg2+', 'SO4_2-', '\PFAS_-', 'Hardness_2+']"
+   "Ion", ":math:`j`", "['Na_+', 'Ca_2+', '\Cl_-', 'Mg2+', 'SO4_2-', '\PFAS_-', 'Hardness_2+'] \  :sup:`1`"
+
+ :sup:`1` "Ion" is a subset of "Component" and uses the same symbol j.
+
+Degrees of Freedom
+------------------
+Aside from the inlet feed state variables (i.e., temperature, pressure, component molar flowrate),
+
+the Ion Exchange (0D) model has at least an additional 27 degrees of freedom that
```

Comment:
Given this may be a large and intimidating number, is it worth adding a clarifying statement that some may be fixed using `.fix()` as in the flowsheet demonstration since its fair to assume default as opposed to needing to know `.fix(value)` for all 27 variables? Though this may be too suggestive, not sure.

Reply:
I edited the note below this table, so let me know if this helps.